Havit.Blazor.Components.Web.Bootstrap/Forms/SearchBox/HxSearchBox.razor.cs

```diff
@@ -201,6 +201,12 @@ public partial class HxSearchBox<TItem> : IAsyncDisposable
 	/// </summary>
 	[Parameter] public RenderFragment InputGroupEndTemplate { get; set; }
 
+	/// <summary>
+	/// If true, the first suggestion is highlighted until another is chosen by the user.
+	/// </summary>
+	[Parameter] public bool? HighlightFirstSuggestion { get; set; }
+	protected bool HighlightFirstSuggestionEffective => this.HighlightFirstSuggestion ?? GetSettings()?.HighlightFirstSuggestion ?? GetDefaults()?.HighlightFirstSuggestion ?? throw new InvalidOperationException(nameof(HighlightFirstSuggestion) + " default for " + nameof(HxSearchBox) + " has to be set.");
+
```

Comment:
I see this is added as new parameter. We will probably remove the parameter completely (in `HxAutosuggest`).